qnexus/client/teams.py

```diff
@@ -54,6 +54,27 @@ def get(name: str) -> TeamRef:
     return teams_list[0]
 
 
+def _fetch(id: str) -> TeamRef:  # pylint: disable=redefined-builtin
```

Comment:
I think having a more explicit name like `_fetch_by_id` would be better. But it's fine if we leave it like it is, since it's a private method.

Reply:
Yeah generally I have just had an internal `_fetch` function for most of the modules so was just keeping with the pattern, renaming it might make sense if we want to make it explicit to qnexus developers I guess